ext/console/01_console.js

```diff
@@ -2742,34 +2742,34 @@ const HSL_PATTERN = new SafeRegExp(
 );
 
 function parseCssColor(colorString) {
-  if (MapPrototypeHas(colorKeywords, colorString)) {
-    colorString = MapPrototypeGet(colorKeywords, colorString);
+  if (colorKeywords.has(colorString)) {
```

Comment:
FYI to other reviewers: safe because `SafeMap`, but may result in `prefer-primordials` linter errors. Let's see :)